### ISSUE TITLE
Install bcmath as part of standard setup; use PHP 7.3 instead of PHP 7.2

### ISSUE THREAD

**Issue by dumblob** [downtownallday/cloudinabox#8]
This is what NC tells admin after fresh install in the web settings interface:

>There are some warnings regarding your setup.
>
>You are currently running PHP 7.2.24-0ubuntu0.18.04.7. Upgrade your PHP version to take advantage of performance and security updates provided by the PHP Group as soon as your distribution supports it.
>
>Nextcloud 20 is the last release supporting PHP 7.2. Nextcloud 21 requires at least PHP 7.3.
>
>This instance is missing some recommended PHP modules. For improved performance and better compatibility it is highly recommended to install them.
>
>    ```
>    bcmath
>    ```

Any plans to solve these?

**Comment by downtownallday** [comment]
With the frequency of Nextcloud releases, php 7.2 isn't going to be viable very much longer. I can look into php 7.3. 7.2 is what comes with Ubuntu Bionic and there is no official support for 7.3. What a pain. Alternatively, one could upgrade ubuntu which comes with 7.4, which is probably what cononical wants.

**Comment by dumblob** [comment]
I feel the pain. I'd say it'd be easiest to:

1. use the newest by Nextcloud supported Ubuntu (as that's what's "guaranteed" to work) and hope the correct PHP etc. comes with it
1. write a huge warning to `cloudinabox` readme, that it requires a **different** version of Ubuntu than MiaB-LDAP
1. check Ubuntu version technically in the installation scripts

**Comment by downtownallday** [comment]
Looks like Bionic has official php 7.4 support.
```
$ apt-cache search php7.4
php7.4 - server-side, HTML-embedded scripting language (metapackage)

```

**Comment by downtownallday** [comment]
Oops, I take it back. I ran that search on a ubuntu 20/focal fossa machine.

**Comment by downtownallday** [comment]
Fixed by v0.03. Reopen or create a new issue if needed.
